src/Constant.js

```diff
@@ -0,0 +1,221 @@
+            { id: 'Stch Mtm', name: t.translate('Stochastic Momentum Index'), description: t.translate('There isn\'t any description here.'), icon: IndicatorStochasticMomentumIcon },
+            { id: 'Williams %R', name: t.translate('William\'s Percent Range'), description: t.translate('There isn\'t any description here.'), icon: IndicatorWilliamPercentIcon },
+        ],
+    },
+    {
+        id: 'trend',
+        name: t.translate('Trend'),
+        icon: IndicatorCatTrendLightIcon,
+        items: [
+            { id: 'Aroon', name: t.translate('Aroon'), description: t.translate('There isn\'t any description here.'), icon: IndicatorAroonIcon },
+            { id: 'ADX', name: t.translate('ADX/DMS'), description: t.translate('There isn\'t any description here.'), icon: IndicatorAdxIcon },
+            { id: 'CCI', name: t.translate('Commodity Channel Index'), description: t.translate('There isn\'t any description here.'), icon: IndicatorCommodityChannelIndexIcon },
+            { id: 'Ichimoku Clouds', name: t.translate('Ichimoku Clouds'), description: t.translate('There isn\'t any description here.'), icon: IndicatorIchimokuIcon },
+            { id: 'PSAR', name: t.translate('Parabolic SAR'), description: t.translate('There isn\'t any description here.'), icon:  IndicatorParabolicIcon },
+            { id: 'ZigZag', name: t.translate('Zig Zag'), description: t.translate('There isn\'t any description here.'), icon: IndicatorZigZagIcon },
+        ],
+    },
+    {
+        id: 'volatility',
+        name: t.translate('Volatility'),
+        icon: IndicatorCatVolatilityIcon,
+        items: [
+            { id: 'Bollinger Bands', name: t.translate('Bollinger Bands'), description: t.translate('There isn\'t any description here.'), icon: IndicatorBollingerIcon },
+            { id: 'Donchian Channel', name: t.translate('Donchian Channel'), description: t.translate('There isn\'t any description here.'), icon: IndicatorDonchianIcon },
+        ],
+    },
+    {
+        id: 'moving-averages',
+        name: t.translate('Moving averages'),
+        icon: IndicatorCatAveragesIcon,
+        items: [
+            { id: 'ma', name: t.translate('Moving Average (MA)'), description: t.translate('There isn\'t any description here.'), icon: IndicatorAveragesIcon },
+            { id: 'MA Env', name: t.translate('Moving Average Envelope'), description: t.translate('There isn\'t any description here.'), icon: IndicatorEnvelopeIcon },
+        ],
+    },
+    {
+        id: 'others',
+        name: t.translate('Others'),
+        icon: IndicatorCatOtherIcon,
+        items: [
+            { id: 'Alligator', name: t.translate('Alligator'), description: t.translate('There isn\'t any description here.'), icon: IndicatorAlligatorIcon },
+            { id: 'Fractal Chaos Bands', name: t.translate('Fractal Chaos Band'), description: t.translate('There isn\'t any description here.'), icon: IndicatorFractalChaosIcon },
+        ],
+    },
+];
```

Comment:
and for all of theses

Reply:
Sure @mahdipourziaei-binary I'll handle these in another card later as these all, are an improvement.